modules/heat_conduction/test/tests/fvbcs/FVFunctorConvectiveHeatFluxBC/fv_functor_convective_heat_flux.i

```diff
@@ -0,0 +1,78 @@
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 5
+    ny = 5
+    xmax = 2
+  []
```

Comment:
rework your test to have multiple blocks. The T_wall and T_inf are likely meant to be defined on different sides of the interface

Reply:
Reworked in next commit, FVFluxBC doesn't work on internal interfaces so I separated it into a multiapp with solid domain in one input and fluid domain in the other, with a nearest node transfer

Reply:
it works on internal to the mesh, external to the domain of definition of a variable